sdlib/d/gc/page.d

```diff
@@ -297,6 +308,7 @@ private:
 		auto ptr = block.address - leadSize;
 		auto npages = pages + extraBlocks * PagesInBlock;
 
+		usedPageCount += npages;
```

Comment:
Same here, you compute all the elements required to update the extent, interleave this, then use them to update the extent.

Reply:
So where should this go? I need `npages`, and it's not calculated until line 309.

the next line returns, so it can't go after that.

Reply:
ha ok, sorry yeah, keep this one here.